assets/sass/components/global/_googlesitekit-publisher-wins.scss

```diff
@@ -165,6 +165,10 @@
 		margin-left: $grid-gap-desktop;
 	}
 
+	.mdc-button + .spinner + .googlesitekit-cta-link {
+		margin-left: $grid-gap-desktop;
+	}
+
```

Comment:
We can simplify this by combining the block here with the one above it and removing the `mdc-button` from the beginning since the `spinner` is rendered unconditionally (the CTA button would never be directly adjacent to the cta link anymore). So really I think the only selector we need for this is `.spinner + .googlesitekit-cta-link`.

Reply:
Good catch - I thought the component was conditional based on `isSaving` but its hidden only via CSS.